pkg/http/router.go

```diff
@@ -66,6 +66,10 @@ func Initialize(r *gin.Engine) {
 		api.GET("/manifests/:account/:location/:kind", core.GetManifest)
 		api.GET("/manifests/:account/:location/:kind/cluster/:application/:cluster/dynamic/:target", core.GetManifestByTarget)
 
+		// Instances API controller.
+		api.GET("/instances/:account/:location/:name", core.GetInstance)
+		api.GET("/instances/:account/:location/:name/console", core.GetInstanceConsole)
+
```

Comment:
The swagger api can be updated to include the instances endpoints

Reply:
Sure. I'll go ahead and do that.